publish-service/src/main/java/com/ericsson/eiffel/remrem/publish/controller/ProducerController.java

```diff
@@ -195,9 +195,9 @@ public ResponseEntity send(
             return send(msgProtocol, userDomain, tag, routingKey, inputBody);
         } catch (JsonSyntaxException e) {
             String exceptionMessage = e.getMessage();
-            log.error("Unexpected exception caught due to parsed json data", exceptionMessage);
+            log.error("Cannot parse the following JSON data: " + body, exceptionMessage);
```

Comment:
Separate the JSON data from the rest of logs:

... data:\n" + body + '\n'

Reply:
Done